wls-gui-wahllokalsystem/src/components/ergebnisermittlung/TheErgebnisermittlungStimmzettelumschlaegeCard.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <v-container>
+    <v-card>
+      <v-card-title>Zählen der Stimmzettel</v-card-title>
+      <v-card-text class="pb-0 pt-2">
+        <v-form
+          ref="anzahlStimmzettelForm"
+          v-model="anzahlStimmzettelValidForm"
+        >
+          <div class="d-flex flex-wrap justify-start">
+            <div>
+              <v-number-input
+                v-model="wahl.stimmzettelumschlaege.anzahlWaehler"
+                class="mr-4"
+                :rules="[REQUIRED, MIN_NUMBER(0), MAX_NUMBER(9999)]"
+                min-width="30rem"
+                data-test="numberInputAnzahlStimmzettel"
+                label="Anzahl der Stimmzettel"
+                clearable
+              />
+            </div>
+          </div>
+        </v-form>
+      </v-card-text>
+      <v-card-actions>
+        <base-button-save
+          active
+          :loading="isStimmzettelumschlaegeSaving"
+          :disabled="isSaveButtonDisabled"
+          @click="onSaveAnzahlStimmzettelClicked"
+        />
+      </v-card-actions>
+    </v-card>
+  </v-container>
+</template>
+
+<script setup lang="ts">
+import { computed, ref } from "vue";
+import {
+  VCard,
+  VCardActions,
+  VCardText,
+  VCardTitle,
+  VContainer,
+  VForm,
+  VNumberInput,
+} from "vuetify/components";
+
+import BaseButtonSave from "@/components/common/buttons/BaseButtonSave.vue";
+import { useWahlenStore } from "@/stores/wahlenStore.ts";
+import { MAX_NUMBER, MIN_NUMBER, REQUIRED } from "@/util/rules.ts";
+
+const props = defineProps<{
+  wahlId: string;
+}>();
+
+const {
+  getWahlOrUndefinedById,
+  saveStimmzettelumschlaege,
+  isStimmzettelumschlaegeSaving,
+} = useWahlenStore();
```

Comment:
isSaving ist ein ref deswegen wird aktuell der loading state nicht angezeigt, weil es falsch importiert wurde
```suggestion
const {
  getWahlOrUndefinedById,
  saveStimmzettelumschlaege,
} = useWahlenStore();
const { isStimmzettelumschlaegeSaving } = storeToRefs(useWahlenStore());
```